lib/transaction/payload/coinbasepayload.js

```diff
@@ -92,11 +104,7 @@ CoinbasePayload.prototype.validate = function() {
  */
 CoinbasePayload.prototype.toJSON = function toJSON() {
   this.validate();
-  return {
-    version: this.version,
-    height: this.height,
-    merkleRootMNList: this.merkleRootMNList
-  };
+  return this;
```

Comment:
Shall we return an object without the prototype and private properties which might be introduced later?

Reply:
done